docs/pages/debugging/tools.mdx

```diff
@@ -205,11 +205,16 @@ Pressing the <kbd>shift</kbd> + <kbd>m</kbd> shortcut will also show installed [
 
 ## React Native Debugger
 
-> **warning** The React Native Debugger does not support apps using Hermes, see the [Chrome DevTools](#debugging-with-chrome-devtools) section.
+> **warning** The React Native Debugger requires Remote JS debugging, which has been deprecated since [React Native 0.73](https://reactnative.dev/docs/other-debugging-methods#remote-javascript-debugging-deprecated).
+> The React Native Debugger is a standalone app that wraps the React DevTools, Redux DevTools, and the Chrome DevTools. Unfortunately, [React Native Debugger requires](https://github.com/jhen0409/react-native-debugger/discussions/774) the deprecated Remote JS debugging workflow and is incompatible with Hermes.
```

Comment:
```suggestion
> **warning** The React Native Debugger requires Remote JS debugging, which has been deprecated since [React Native 0.73](https://reactnative.dev/docs/other-debugging-methods#remote-javascript-debugging-deprecated).

The React Native Debugger is a standalone app that wraps the React DevTools, Redux DevTools, and the Chrome DevTools. Unfortunately, it requires the [deprecated Remote JS debugging workflow](https://github.com/jhen0409/react-native-debugger/discussions/774) and is incompatible with Hermes.
```

The first warning and the second part are repetitive. The second sentence should be an introduction to this paragraph instead of a part of the warning. Also, let's not repeat " React Native Debugger" in every sentence in the same paragraph. This also allows the subject of the last sentence to be linked correctly.